cycle-scripts-es-browserify/package.json

```diff
@@ -19,6 +19,7 @@
     "cycle-scripts": "./index.js"
   },
   "dependencies": {
+    "ava": "^0.17",
```

Comment:
You see inflexibility where I see openness. There isn't the need to duplicate each core flavor, because you could create new ones and publish them on npm independently.

Each `cycle-scripts-*` you see here, could be copied into `motorcycle-scripts-*` (or forked) and customized for motorcycle (you could just ignore cycle-specific options on init script). There you could change test libs, template files and everything else you see suitable as motorcycle defaults.

I'm not rejecting changes on the tool, I'm just arguing that there's easier ways to achieve the same results without bloating the core tool and flavors.

Ok, so, I see X valuable proposals until now:

- Including `@cycle/isolate` as a default dependency
- Adding motorcycle flavors
- Making the test library/tool an option
- Templating files

My proposal is that we turn each of this points into issues and build them progressively. What do you think?